pkgs/games/r2modman/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib
+, mkYarnPackage
+, fetchFromGitHub
+, gitUpdater
+, makeWrapper
+, makeDesktopItem
+, electron
+}:
+
+let
+  execName = "r2modman";
+in
+mkYarnPackage rec {
+  name = "r2modmann";
+  version = "3.1.41";
+
+  src = fetchFromGitHub {
+    owner = "ebkr";
+    repo = "r2modmanPlus";
+    rev = "v${version}";
+    hash = "sha256-nnmzIpV+Rn2zBJDBUehcUpvh3U4jqGLLGoIoxlZy6SA=";
+  };
+
+  packageJSON = ./package.json;
+  yarnLock = ./yarn.lock;
+  yarnNix = ./yarn.nix;
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  DISABLE_ESLINT_PLUGIN = "true";
+  # Workaround for webpack bug
+  # https://github.com/webpack/webpack/issues/14532
+  NODE_OPTIONS = "--openssl-legacy-provider";
+
+  postBuild =
+    let
+      yarnCmd = "yarn --offline --frozen-lockfile --ignore-engines --ignore-scripts --lockfile ${yarnLock}";
+    in
+    ''
+      ${yarnCmd} quasar build --mode electron --skip-pkg
+    '';
+
+  # Disable bundling into a tar archive.
+  doDist = false;
+
+  postInstall = ''
+    makeWrapper "${electron}/bin/electron" "$out/bin/${execName}" \
+      --inherit-argv0 \
+      --add-flags "$out/libexec/r2modman/deps/r2modman/dist/electron/UnPackaged"
+  '';
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = "r2modman";
+      exec = "${execName}";
+      icon = "${placeholder "out"}/icon.png";
+      desktopName = "r2modman";
+      comment = meta.description;
+      categories = [ "Game" ];
+      keywords = [ "launcher" "mod manager" "thunderstore" ];
+    })
+  ];
```

Comment:
Don't you need the `copyDesktopItems` hook in `nativeBuildInputs` for this to do anything?